firefox-ios/Client/Application/AccessibilityIdentifiers.swift

```diff
@@ -19,7 +19,7 @@ public struct AccessibilityIdentifiers {
         static let urlBarBorder = "TabToolbar.urlBarBorder"
         static let settingsMenuButton = "TabToolbar.menuButton"
         static let homeButton = "TabToolbar.homeButton"
-        static let trackingProtection = "TabLocationView.trackingProtectionButton"
+        static let trackingProtection = "AddressToolbar.lockIcon"
```

Comment:
We should use the new a11y id for the lock icon (`AccessibilityIdentifiers.Browser.AddressToolbar.lockIcon`) instead of renaming this one which is only used for the old toolbar.

Reply:
updated